.github/workflows/ci_cd.yml

```diff
@@ -110,12 +110,94 @@ jobs:
     name: "Check library vulnerabilities"
     runs-on: ubuntu-latest
     steps:
-      - uses: ansys/actions/check-vulnerabilities@v8.1
+      - uses: ansys/actions/check-vulnerabilities@feat/allow-for-extra-targets-in-vulnerabilities-checks
         with:
           python-version: ${{ env.MAIN_PYTHON_VERSION }}
           token: ${{ secrets.PYANSYS_CI_BOT_TOKEN }}
           python-package-name: 'ansys-acp-core'
           dev-mode: ${{ github.ref != 'refs/heads/main' }}
+          extra-targets: "plotting examples"
```

Comment:
```suggestion
          extra-targets: "plotting"
```

We only need the `plotting` extra here, but added two extras for the purposes of testing the action.